xcm-bridge/src/mock/mod.rs

```diff
@@ -0,0 +1,362 @@
+// The Licensed Work is (c) 2022 Sygma
```

Comment:
Don't forget to use the latest code from [here](https://github.com/paritytech/polkadot-sdk/tree/master/polkadot/xcm/xcm-simulator/example/src), not necessary though